CHANGELOG.md

```diff
@@ -25,6 +25,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Broker UI Docker Image: `ghcr.io/sovity/edc-ui:{{ UI_VERSION }}`
 - Sovity EDC CE: {{ CE Release Link }}
 
+## [v4.1.0] - 2024-04-02
+
+### Overview
+
+Pull changes from EDC CE 7.3.0 into the broker.
+
+### Detailed Changes
+
+#### Minor
+
+- Bumped EDC version to 7.3.0:
+  - UIAsset: Replaced unsafe additional and private properties with safer alternative fields customJsonAsString (not affected by Json LD manipulation) and customJsonLdAsString (affected by Json LD manipulation), along with their private counterparts.
```

Comment:
This change was done in the EDC CE, you should write "Broker UI: Support for UIAsset's customJsonAsString and customJsonLdAsString, along with their private counterparts"